### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by alex-zenml**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
Update CLI main docstring to cover `zenml integration` and `zenml container-registry` commands

### CODE REVIEW

**Review by AlexejPenner — APPROVED** [review]
:)

**Review by schustmi — APPROVED** [review]

File: src/zenml/cli/__init__.py [hunk 1]
```diff
@@ -165,6 +165,43 @@
 
    zenml example run quickstart
 
+Using integrations
+------------------
+
+Integrations are the different pieces of a project stack that enable custom
+functionality. This ranges from bigger libraries like `kubeflow` for
```
Comment (htahir1) [inline src/zenml/cli/__init__.py:172]: maybe we can link these?

File: src/zenml/cli/__init__.py [hunk 2]
```diff
@@ -247,6 +284,46 @@
 
    zenml orchestrator delete ORCHESTRATOR_NAME
 
+Customizing your Container Registry
+-----------------------------------
+
+The container registry is where all the containers that are used by a
```
Comment (htahir1) [inline src/zenml/cli/__init__.py:290]: I think we should rather say 'images' than containers... where all the images that are used by a ...

**Review by htahir1 — COMMENTED** [review]
Cool! Few comments

**Comment by alex-zenml** [comment]
@htahir1 addressed your two suggestions above and fixed some other references in `examples` that I had forgotten to check.

**Review by htahir1 — APPROVED** [review]
Awesome! Thanks for the update!

**Pull request merged**